examples/opentelemetry/docker-compose.yaml

```diff
@@ -63,9 +63,11 @@ services:
     healthcheck:
       test: ["CMD-SHELL", "curl -sf http://localhost:13133 || exit 1"]
       interval: 1s
-      timeout: 120s
+      timeout: 10s
       retries: 120
-      start_period: 5s
-    command: ["--config=/etc/otel-collector-config.yaml"]
+      start_period: 2s
     ports:
+    - "4137:4137" # gRPC port
     - "${PORT_UI:-55679}:55679"
+    volumes:
+    - ./otel-collector-config.yaml:/etc/otel-collector-config.yaml
```

Comment:
this is unnecessary and breaks CI - the examples dont allow you to mount Envoy configs from the filesystem - the reason for this is that some devs work on a filesystem with strict `umask` set

Reply:
Ok

Reply:
just realized this isnt an envoy config but i think the same applies - it will need to be loaded in the docker build

Reply:
Done in 4c4e27eab5